Tests/UnitTests/Core/EventData/CMakeLists.txt

```diff
@@ -5,5 +5,4 @@ add_unittest(FreeTrackParameters FreeTrackParametersTests.cpp)
 add_unittest(MeasurementHelpers MeasurementHelpersTests.cpp)
 add_unittest(Measurement MeasurementTests.cpp)
 add_unittest(MultiTrajectory MultiTrajectoryTests.cpp)
-add_unittest(ParameterSet ParameterSetTests.cpp)
-add_unittest(TrackState TrackStateTests.cpp)
+add_unittest(ParameterSet ParameterSetTests.cpp)
```

Comment:
```suggestion
add_unittest(ParameterSet ParameterSetTests.cpp)

```

Reply:
?

Reply:
Could you explain why a blank line should be added?

Reply:
This was a missing newline at the end of the file. This is purely aesthetically. Feel free to ignore it.